src/NServiceBus.AcceptanceTests/Core/Installers/When_creating_queues.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace NServiceBus.AcceptanceTests.Installers
```

Comment:
Since this is in the Core folder, should be `NServiceBus.AcceptanceTests.Core.Installers`